pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -522,33 +572,33 @@ details {
 
 // Forms
 form {
-
   label {
     display: block;
     text-align: left;
     font-size: 1rem;
     font-weight: var(--semi-bold-font-weight);
   }
 
-  input[type="text"], select {
+  input[type="text"],
+  select {
     display: block;
     width: 100%;
-    margin: .25rem auto 1.5rem;
+    margin: 0.25rem auto 1.5rem;
     padding: 0.5rem;
     border: 1px solid;
     border-radius: $border-radius;
     font-family: var(--main-font);
     font-weight: var(--regular-font-weight);
     font-size: 1rem;
-    transition: all ease-in-out .2s;
+    transition: all ease-in-out 0.2s;
 
     &:focus {
       @include focus-outline;
     }
   }
 
   button,
-  &[type=submit] {
+  &[type="submit"] {
```

Comment:
I'm surprised by this kind of linter fix- where something would go from not in quotes, to in quotes

Reply:
I think it should be in quotes, but also works without?? ([ref](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors), also wow there are so many ways to use the attribute selector 🙀)